build.gradle

```diff
@@ -67,6 +67,10 @@ ext {
 
     kotlinCoroutinesVersion = '1.6.0'
     kotlinSerializationVersion = '1.3.2'
+    coreKtxVersion = '1.7.0'
+    composeVersion = '1.1.0'
+    composeConstraintLayout = '1.0.0'
+    composeActivityVersion = '1.4.0'
```

Comment:
`androidx.activity:activity-compose` is a submodule of the `androidx.activity` with the same version as `androidxActivityVersion`above: https://developer.android.com/jetpack/androidx/releases/activity
Where is `coreKtxVersion` used? That's the same as `androidxCoreVersion`.
Where is `composeVersion` used? That's the same as `androidxComposeVersion`.
`composeActivityVersion` seems to be the only new one needed, because it's a different version than the `androidxConstraintlayoutVersion` above: https://developer.android.com/jetpack/androidx/releases/constraintlayout
Let's follow the same pattern for naming, and make it `androidxConstraintlayoutComposeVersion`, list it above in lexicographical order.

Reply:
fixed. idk what happened when I merged.